plugins/inputs/mongodb/mongodb_data.go

```diff
@@ -31,28 +31,43 @@ func NewMongodbData(statLine *StatLine, tags map[string]string) *MongodbData {
 }
 
 var DefaultStats = map[string]string{
+	"inserts_cnt":               "InsertCnt",
```

Comment:
Let's name the fields without the `_cnt` where possible, trying to stick to the MongoDB naming (but still in snake_case), as listed in mongostat.go.  So this would be `insert` instead of `inserts`.

For the fields which would collide with the existing field, such as `cursor_timed_out`, then the suffix will need to be added but write it out like `cursor_timed_out_count`.

Reply:
I renamed the added metrics.